tests/clock/test_clock.py

```diff
@@ -0,0 +1,427 @@
+import logging
+import random
+import string
+import pytest
+import time
+import allure
+import datetime as dt
+
+from contextlib import contextmanager
+from tests.common.errors import RunAnsibleModuleFail
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.sanity_check(skip_sanity=True),
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.skip_check_dut_health,
+    pytest.mark.clock
+]
+
+
+@contextmanager
```

Comment:
Cool, should be placed in a utility file, rather than the test file. Afaik such context manager already exists, check with @nhe-NV , there's no need to duplicate it.

Reply:
done. now i'm using the existing allure_step_wrapper from tests.common.plugins.allure_wrapper